<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(109, 197, 108);
font: 75 italic 14pt &quot;URW Chancery L&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>-50</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Добавление нового рецепта</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>337</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>URW Chancery L</family>
           <pointsize>14</pointsize>
           <weight>9</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Выберите кухню:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cousine_cmBx">
         <property name="currentText">
          <string>Кухня..</string>
         </property>
         <item>
          <property name="text">
           <string>Кухня..</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Украинская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Русская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Узбекская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Китайская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ирландская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Итальянская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Корейская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Грузинская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Японская</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Французкая</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>URW Chancery L</family>
           <pointsize>14</pointsize>
           <weight>9</weight>
           <italic>true</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Выберите основу блюда:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="basis_cmBx">
         <item>
          <property name="text">
           <string>Основа..</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Мучное</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Овощи</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Рыба</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Мясо</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="category_grBx">
       <property name="font">
        <font>
         <family>URW Chancery L</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Выберите категорию:</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>331</width>
          <height>107</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QCheckBox" name="drinks_chBx">
           <property name="text">
            <string>Напитки</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="salad_chBx">
           <property name="text">
            <string>Салаты</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="bakery_chBx">
           <property name="text">
            <string>Выпечка</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="snak_chBx">
           <property name="text">
            <string>Закуски</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="dessert_chBx">
           <property name="text">
            <string>Десерты</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="second_dish_chBx">
           <property name="text">
            <string>Вторые блюда</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="sauce_chBx">
           <property name="styleSheet">
            <string notr="true">font: 75 italic 13pt &quot;URW Chancery L&quot;;</string>
           </property>
           <property name="text">
            <string>Соусы</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="first_dish_chBx">
           <property name="text">
            <string>Первые блюда</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="dough_chBx">
           <property name="text">
            <string>Тесто</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="method_grBx">
       <property name="font">
        <font>
         <family>URW Chancery L</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Выберите способ приготовления:</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>331</width>
          <height>107</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QCheckBox" name="bake_chBx">
           <property name="text">
            <string>Запекать</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="stew_chBx">
           <property name="text">
            <string>Тушить</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="boil_chBx">
           <property name="text">
            <string>Варить</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="fry_chBx">
           <property name="text">
            <string>Жарить</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cut_chBx">
           <property name="text">
            <string>Нарезать</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="event_grBx">
       <property name="font">
        <font>
         <family>URW Chancery L</family>
         <pointsize>14</pointsize>
         <weight>9</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Выберите событие:</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>331</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="new_year_chBx">
           <property name="text">
            <string>Новый год</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="christmas_chBx_2">
           <property name="text">
            <string>Рецепт на каждый день</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="christmas_chBx">
           <property name="text">
            <string>Рождество</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="easter_chBx">
           <property name="text">
            <string>Пасха</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="thanks_give_day_chBx">
           <property name="text">
            <string>День благодарения</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="thanks_give_day_chBx_2">
           <property name="text">
            <string>Пост</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="recipes_tblWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>741</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(109, 197, 108);
font: 75 italic 20pt &quot;URW Chancery L&quot;;</string>
    </property>
    <property name="text">
     <string>Рецепты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_Btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>710</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск по рецептам</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="show_add_recipe_page_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(109, 197, 108);
font: 75 italic 18pt &quot;URW Chancery L&quot;;</string>
       </property>
       <property name="text">
        <string>Добавить новый рецепт</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_select_recipe">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(58, 77, 58);
font: 75 italic 18pt &quot;URW Chancery L&quot;;</string>
       </property>
       <property name="text">
        <string>Сортировка рецептов за категорией</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_sort_recipe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(109, 197, 108);
font: 75 italic 18pt &quot;URW Chancery L&quot;;</string>
       </property>
       <property name="text">
        <string>Сортировка рецептов по цене</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
